style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.top-header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.icons-section {
+  padding: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.time {
+  margin-left: 10px;
+}
+
+.menu-icon {
+  margin-right: 10px;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 20px;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.logo-name {
+  font-family: 'Roboto', Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  size: 18px;
+  line-height: 24px;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+img {
+  display: block;
+}
+
+.menu-bar {
+  display: flex;
+  flex-direction: column;
+  gap: -5px;
+}
+
+.headline {
+  font-family: 'Poppins', Courier, monospace;
+  font-size: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 8px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background: #1c1a19;
+  color: white;
+  box-sizing: border-box;
+  background-image: url('images/Illustration-header.png');
+  background-repeat: no-repeat;
+  background-size: 275px;
+  background-position: bottom -70px right -125px;
+}
+
+.main-header span {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 64px;
+  line-height: 64px;
+  letter-spacing: 0.37;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.main-header h2 {
+  font-size: 24px;
+}
+
+.headline-desc {
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.social-icons {
+  flex: none;
+  order: 2;
+  flex-grow: 0;
```

Comment:
- [ ] The styling here should be the styles of the list of different icons not just a copied image from the design.